geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -17,25 +17,22 @@ limitations under the License.
 
 <% set_title(product_name, "for Redis") %>
 
-<%=vars.product_name%> for Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
+<%=vars.product_name%> for Redis allows <%=vars.product_name%> to function as a
 highly-available Redis data store, letting Redis applications take advantage of
-<%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
-server in the same way they connect to a Redis server, using a hostname and a port number, with
-optional password authentication.
+<%=vars.product_name%>’s scaling capabilities with minimal changes to their client code.
 
-<img src="../images_svg/geode_for_redis.svg" class="image" />
```

Comment:
The image file itself should also be removed (as Hale pointed out in Slack).

Reply:
Removed in recent update.  Thank you for catching this!